src/components/App.js

```diff
@@ -62,10 +58,12 @@ class App extends Component {
       <section className="App">
         <section className="text-source-container">
           <h2>new route: (?) section to select text source</h2>
+          <button className="App-speechBtn" onClick={() => this.setState({speechEnabled: !this.state.speechEnabled})}>{!this.state.speechEnabled ? icons.micOff : icons.micOn }</button>
```

Comment:
So this button is present on the screen the whole time and enables speech? Works for me, I'll figure out a way to enable it 100% of the time for conditional rendering too.